proto/pulumi/language.proto

```diff
@@ -133,6 +133,9 @@ message RunRequest {
     string organization = 12;       // the organization of the stack being deployed into.
     google.protobuf.Struct configPropertyMap = 13; // the configuration variables to apply before running.
     ProgramInfo info = 14; // the program info to use to execute the program.
+
+    // The target of a codegen.LoaderServer to use for loading schemas.
+    string loader_target = 15;
```

Comment:
The Go code should match, but we already consistently use "address" for the other things in the Go code. 
AFAIK "target" is more correct as far as grpc terminology goes, but might be a bit painful changing all the Go code to use Target instead of address because some of its in sdk/go/common.